<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.1 (Python 3.8.3 on win32)" generated="20200612 09:45:51.429" rpa="false">
<suite id="s1" name="App Member" source="D:\jiangbm\XFJ\API\App_Member">
<suite id="s1-s1" name="UploadPortrait" source="D:\jiangbm\XFJ\API\App_Member\UploadPortrait.robot">
<kw name="Create Session Common" library="Common" type="setup">
<kw name="Fapi Create Session">
<arguments>
<arg>${APPIF_ALIAS}</arg>
<arg>${APPIF_URL}</arg>
</arguments>
<msg timestamp="20200612 09:45:51.954" level="FAIL">No keyword with name 'Fapi Create Session' found. Did you mean:
    RequestsLibrary.Create Session</msg>
<status status="FAIL" starttime="20200612 09:45:51.954" endtime="20200612 09:45:51.954"></status>
</kw>
<status status="FAIL" starttime="20200612 09:45:51.947" endtime="20200612 09:45:51.955"></status>
</kw>
<test id="s1-s1-t1" name="无效会员上传头像，接口返回失败">
<tags>
<tag>会员接口/上传会员头像（孙安）</tag>
<tag>作者：江宝敏</tag>
<tag>冒烟集-新福建APP</tag>
</tags>
<status status="FAIL" starttime="20200612 09:45:51.955" endtime="20200612 09:45:51.957" critical="yes">Parent suite setup failed:
No keyword with name 'Fapi Create Session' found. Did you mean:
    RequestsLibrary.Create Session</status>
</test>
<kw name="Fapi Delete All Sessions" type="teardown">
<msg timestamp="20200612 09:45:51.964" level="FAIL">No keyword with name 'Fapi Delete All Sessions' found. Did you mean:
    RequestsLibrary.Delete All Sessions</msg>
<status status="FAIL" starttime="20200612 09:45:51.964" endtime="20200612 09:45:51.964">No keyword with name 'Fapi Delete All Sessions' found. Did you mean:
    RequestsLibrary.Delete All Sessions</status>
</kw>
<doc>上传会员头像</doc>
<status status="FAIL" starttime="20200612 09:45:51.460" endtime="20200612 09:45:51.964">Suite setup failed:
No keyword with name 'Fapi Create Session' found. Did you mean:
    RequestsLibrary.Create Session

Also suite teardown failed:
No keyword with name 'Fapi Delete All Sessions' found. Did you mean:
    RequestsLibrary.Delete All Sessions</status>
</suite>
<status status="FAIL" starttime="20200612 09:45:51.432" endtime="20200612 09:45:51.965"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1">会员接口/上传会员头像（孙安）</stat>
<stat pass="0" fail="1">作者：江宝敏</stat>
<stat pass="0" fail="1">冒烟集-新福建APP</stat>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="App Member">App Member</stat>
<stat pass="0" fail="1" id="s1-s1" name="UploadPortrait">App Member.UploadPortrait</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20200612 09:45:51.465" level="ERROR">Error in file 'D:\jiangbm\XFJ\API\App_Member\MemberCommon.robot' on line 10: Setting 'Force Tags' is not allowed in resource file.</msg>
<msg timestamp="20200612 09:45:51.472" level="ERROR">Error in file 'D:\jiangbm\XFJ\API\Common\Common.robot' on line 3: Importing test library 'FATL.FapiKeywords' failed: ModuleNotFoundError: No module named 'FATL'
Traceback (most recent call last):
  None
PYTHONPATH:
  D:\jiangbm\pycharm\API\Scripts\robot.exe
  D:\Python38-32\python38.zip
  D:\Python38-32\DLLs
  D:\Python38-32\lib
  D:\Python38-32
  d:\jiangbm\pycharm\api
  d:\jiangbm\pycharm\api\lib\site-packages
  D:\Python38-32\lib\site-packages</msg>
<msg timestamp="20200612 09:45:51.885" level="ERROR">Error in file 'D:\jiangbm\XFJ\API\App_Member\MemberCommon.robot' on line 6: Importing test library 'D:\jiangbm\XFJ\API\App_Member\Upload.py' failed: ModuleNotFoundError: No module named 'Secret'
Traceback (most recent call last):
  File "D:\jiangbm\XFJ\API\App_Member\Upload.py", line 6, in &lt;module&gt;
    from Secret import authen,XFJ_Des
PYTHONPATH:
  D:\jiangbm\pycharm\API\Scripts\robot.exe
  D:\Python38-32\python38.zip
  D:\Python38-32\DLLs
  D:\Python38-32\lib
  D:\Python38-32
  d:\jiangbm\pycharm\api
  d:\jiangbm\pycharm\api\lib\site-packages
  D:\Python38-32\lib\site-packages</msg>
<msg timestamp="20200612 09:45:51.890" level="ERROR">Error in file 'D:\jiangbm\XFJ\API\App_Member\MemberCommon.robot' on line 7: Importing test library 'D:\jiangbm\XFJ\API\App_Member\Upload.py' failed: ModuleNotFoundError: No module named 'Secret'
Traceback (most recent call last):
  File "D:\jiangbm\XFJ\API\App_Member\Upload.py", line 6, in &lt;module&gt;
    from Secret import authen,XFJ_Des
PYTHONPATH:
  D:\jiangbm\pycharm\API\Scripts\robot.exe
  D:\Python38-32\python38.zip
  D:\Python38-32\DLLs
  D:\Python38-32\lib
  D:\Python38-32
  d:\jiangbm\pycharm\api
  d:\jiangbm\pycharm\api\lib\site-packages
  D:\Python38-32\lib\site-packages</msg>
<msg timestamp="20200612 09:45:51.946" level="ERROR">Error in file 'D:\jiangbm\XFJ\API\App_Member\MemberCommon.robot' on line 9: Importing test library 'D:\jiangbm\XFJ\API\Secret\authen.py' failed: ModuleNotFoundError: No module named 'Secret'
Traceback (most recent call last):
  File "D:\jiangbm\XFJ\API\Secret\authen.py", line 3, in &lt;module&gt;
    from Secret import XFJ_Des
PYTHONPATH:
  D:\jiangbm\pycharm\API\Scripts\robot.exe
  D:\Python38-32\python38.zip
  D:\Python38-32\DLLs
  D:\Python38-32\lib
  D:\Python38-32
  d:\jiangbm\pycharm\api
  d:\jiangbm\pycharm\api\lib\site-packages
  D:\Python38-32\lib\site-packages</msg>
</errors>
</robot>
